x/staging/posts/keeper/msgs_server_test.go

```diff
@@ -198,6 +198,8 @@ func (suite *KeeperTestSuite) TestMsgServer_CreatePost() {
 			suite.SetupTest()
 			suite.k.SetParams(suite.ctx, types.DefaultParams())
 
+			_ = suite.sk.SaveSubspace(suite.ctx, suite.testData.subspace, suite.testData.postOwner)
```

Comment:
Can we not ignore the error please? I think it's always worth to not ignore them: 

```suggestion
			err := suite.sk.SaveSubspace(suite.ctx, suite.testData.subspace, suite.testData.postOwner)
			suite.Require().NotErr(err)
```

This also applies to all other places